.github/workflows/pythonpackage.yml

```
@@ -7,32 +7,20 @@ env:
 
 jobs:
   codegen:
-    runs-on: ubuntu-latest
+    runs-on:  ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: ["3.8", "3.12"]
+        os: [ "ubuntu-latest", "windows-latest", "macos-latest" ]
```

it would be great if we could keep testing with the oldest supported python version as well, i.e. something like

```suggestion
        os: [ "ubuntu-latest", "windows-latest", "macos-latest" ]
        python-version: ["3.8", "3.13"]
```

(not sure if github-actions likes this...)

---

Done